onnx/reference/ops/op_deform_conv.py

```diff
@@ -0,0 +1,176 @@
+# SPDX-License-Identifier: Apache-2.0
+# pylint: disable=R0912,R0913,R0914,R0915,R1702,W0221
+
+import numpy as np
+
+from onnx.reference.op_run import OpRun
+
+
+def _conv_implementation(  # type: ignore
```

Comment:
Naive question: is this an implementation of deform_conv (if so, better to rename it accordingly), or is it a common implementation (that subsumes conv also)?

Reply:
Thanks for catching. I renamed it to `_deform_conv_implementation`.